classes/Learnosity/Plugin.php

```diff
@@ -34,28 +39,62 @@ public function enqueue_scripts()
 		$lrn_reports_api_url = get_option('lrn_reports_api_url','https://reports-va.learnosity.com/?v1');
 		$lrn_author_api_url = get_option('lrn_author_api_url','https://authorapi-or.learnosity.com/?v1');
 		wp_enqueue_script(
-			'learnosity-items',
-			$lrn_items_api_url,
-			array(),
-			null,
-			false
+		    'learnosity-items',
+            $lrn_items_api_url,
```

Comment:
I think the indentation change here was checked in by accident? It looked correct before [here](https://github.com/Learnosity/wordpress-learnosity/blob/master/classes/Learnosity/Plugin.php#L36)

Reply:
Indentation before was partially not with spaces, fixed it in new commit